spec/requests/users_spec.rb

```diff
@@ -2,9 +2,34 @@
 
 RSpec.describe 'Users', type: :request do
   describe 'GET /index' do
-    it 'returns http success' do
-      get '/users/index'
-      expect(response).to have_http_status(:success)
+    before(:each) do
+      get '/users'
+    end
+
+    it 'Users index is successful' do
+      expect(response).to have_http_status(:ok)
+    end
+    it 'Renders template correctly' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'Response should include right placeholder text' do
+      expect(response.body).to include('List of Users')
+    end
+  end
+
+  describe 'Show single user details' do
+    before(:each) do
+      get '/users/1'
+    end
+    it 'Response will be successful.' do
+      expect(response).to have_http_status(:ok)
+    end
+    it 'Template render correctly' do
+      expect(response).to render_template(:show)
+    end
+    it 'Response should include right placeholder text.' do
+      expect(response.body).to include('User details will be shown here')
```

Comment:
- [ ] Kindly make sure that tests written for index and show actions for User and Post controllers are passing, the tests are currently failing see screenshot above ⬆️